lib/compiler/ops.ts

```diff
@@ -134,6 +134,7 @@ StreamOp.Timer = TimerStreamOp;
 export class AtTimerStreamOp extends StreamOp {
     constructor(public time : Ast.Value[],
                 public expiration_date : Ast.Value|null,
+                public once : Ast.Value|null,
                 public ast : Ast.Stream) {
```

Comment:
You will also need to add code in ops-to-jsir to make use of this field, compiling it to a call to ExecEnvironment.invokeAtTimer with the right parameters.

Reply:
I see, just added it